cypress/e2e/specs/workshop/level.3.cy.js

```diff
@@ -7,5 +7,21 @@ describe('EMON', () => {
 
   it('...', () => {
       // Level Three
+      cy.Login();
+      cy.waitForReact();
+      cy.visit('https://mercadolibre.com.ar/publicaciones/editor-masivo');
+      cy.waitForReact();
+      cy.contains('Filtros');
```

Comment:
```suggestion
      cy.contains('Filtros').click();
```
Desde aquí se puede hacer click